GitUI/Editor/FindAndReplaceForm.cs

```diff
@@ -503,14 +535,18 @@ public void SetScanRegion(int offset, int length)
             _region = new TextMarker(offset, length, TextMarkerType.SolidBlock,
                                      Globals.HalfMix(bkgColor, Color.FromArgb(160, 160, 160)));
             _document.MarkerStrategy.AddMarker(_region);
+            _document.TextContentChanged += DocumentOnTextContentChanged;
+            ScanRegionChanged?.Invoke(this, null);
         }
 
         public void ClearScanRegion()
         {
             if (_region != null)
             {
                 _document.MarkerStrategy.RemoveMarker(_region);
+                _document.TextContentChanged -= DocumentOnTextContentChanged;
                 _region = null;
+                ScanRegionChanged?.Invoke(this, null);
```

Comment:
Never send `null` for event args, this may lead to NRE
```suggestion
                ScanRegionChanged?.Invoke(this, EventArgs.Empty);
```

Reply:
Done